### DISCUSSION TITLE
Girlscript contribution

### DISCUSSION THREAD

**Discussion by bishal7679** [arpit456jain/Cool-Front-End-Templates#192] [General]
can I add my template in your repo for girlscript ?

**Comment by arpit456jain** [comment]
yes u can just wait for some days

**Reply by xoshiraghavx** [reply]
Hello how can I talk to project manager?

**Reply by xoshiraghavx** [reply]
I'm new here, how can I select a project. Where do I need to inform?

**Reply by arpit456jain** [reply]
contact to mentors in discord server

**Comment by xoshiraghavx** [comment]
In which server? General server?

**Reply by TG922** [reply]
Server Name = GSSOC'22 (Girlscript Summer of Code 2022)
